web/tsconfig-serve.json

```diff
@@ -2,6 +2,8 @@
   "compilerOptions": {
     "sourceMap": true,
     "declaration": false,
+    "allowSyntheticDefaultImports": true,
```

Comment:
Shouldn't this go to `tsconfig.base.json`? Why added only to serve?

Reply:
Moved to tsconfig.base.json

https://github.com/Microsoft/TypeScript/issues/28898 - I concede on reload